components/theme/constants.ts

```diff
@@ -1,3 +1,5 @@
+export const SMALL_SCREEN_THRESHOLD = 720; // average mobile screen height in pixels.
```

Comment:
nit: for later, but we can use https://chakra-ui.com/docs/styled-system/responsive-styles for this and use something like `display={{sm: 'hidden', md: 'flex'}}`